src/api/structures/message.ts

```diff
@@ -32,7 +33,8 @@ import { Role } from "./role.ts";
 
 const baseMessage: Partial<Message> = {
   get channel() {
-    return cache.channels.get(this.channelID!);
+    return cache.channels.get(this.channelID!) ||
+      cache.channels.get(this.author?.id!);
```

Comment:
Can you add a note to the JSDoc comment block for this header regarding the change?